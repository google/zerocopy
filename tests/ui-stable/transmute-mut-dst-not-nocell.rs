../ui-nightly/transmute-mut-dst-not-nocell.rs