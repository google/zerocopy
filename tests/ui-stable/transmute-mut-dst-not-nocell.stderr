error[E0277]: the trait bound `Dst: Immutable` is not satisfied
  --> tests/ui-stable/transmute-mut-dst-not-nocell.rs:24:37
   |
24 | const DST_NOT_IMMUTABLE: &mut Dst = transmute_mut!(&mut Src);
   |                                     ^^^^^^^^^^^^^^^^^^^^^^^^
   |                                     |
   |                                     the trait `Immutable` is not implemented for `Dst`
   |                                     required by a bound introduced by this call
   |
   = help: the following other types implement trait `Immutable`:
             bool
             char
             isize
             i8
             i16
             i32
             i64
             i128
           and $N others
note: required by a bound in `AssertDstIsImmutable`
  --> tests/ui-stable/transmute-mut-dst-not-nocell.rs:24:37
   |
24 | const DST_NOT_IMMUTABLE: &mut Dst = transmute_mut!(&mut Src);
   |                                     ^^^^^^^^^^^^^^^^^^^^^^^^ required by this bound in `AssertDstIsImmutable`
   = note: this error originates in the macro `transmute_mut` (in Nightly builds, run with -Z macro-backtrace for more info)
